packages/client/src/entities/theme/model/selectors.ts

```diff
@@ -0,0 +1,4 @@
+import { RootState } from '@@app/appStore';
+
+const selectThemeModule = (state: RootState) => state.theme;
+export const selectCurrentTheme = (state: RootState) => selectThemeModule(state).theme;
```

Comment:
```
export const selectCurrentTheme = (state: RootState) => selectThemeModule(state).theme;
```

Имхо createSelector выгдядит красивее, т.к. виден "поток данных" и мы используем аргумент, а не результат выполнения функции через точку(это немного не очевидно при чтении кода).

Лучше писать результат в переменную.
Либо как я через createSelector, либо хотя бы так:
```suggestion
export const selectCurrentTheme = (state: RootState) => {
    const themeState = selectThemeModule(state);
    return themeState.theme;
 };
```